components/suggest/src/schema.rs

```diff
@@ -6,7 +6,7 @@
 use rusqlite::{Connection, Transaction};
 use sql_support::open_database::{self, ConnectionInitializer};
 
-pub const VERSION: u32 = 5;
+pub const VERSION: u32 = 6;
```

Comment:
Nice, thanks!

Remember to update `upgrade_from`, too, please 😁

Reply:
🙈